build/media_source/plg_system_shortcut/js/shortcut.es6.js

```diff
@@ -0,0 +1,64 @@
+class Shortcut {
+    constructor() {
+      if (!Joomla) {
+        throw new Error('Joomla API is not properly initialised');
+      }
+  
+      ((window, document, Joomla) => {
```

Comment:
`constructor` is only used to initialize the values or methods going to be used in that class object.

Remove all the methods from `constructor` 


<img width="375" alt="Screenshot 2021-06-27 at 9 35 26 AM" src="https://user-images.githubusercontent.com/30978328/123532483-0311dd00-d72b-11eb-9070-c888a46a1eed.png">

Your class should look like this

https://github.com/joomla-projects/gsoc21_accessibility/pull/7#issuecomment-864523757
Follow this code